raster/r.out.gdal/tests/test_r_out_gdal.py

```diff
@@ -48,13 +48,16 @@ def test_basic_cog_export(simple_raster_map, file_format):
     mapname, session, tmp_path = simple_raster_map
     tools = Tools(session=session)
     suffix = FORMAT_DICT[file_format]
-    output_file = tmp_path / f"{mapname}.{suffix}"
-    tools.r_out_gdal(
+    output_file = tmp_path / f"{mapname}_{file_format}.{suffix}"
+    export = tools.r_out_gdal(
```

Comment:
`AttributeError: 'NoneType' object has no attribute 'stderr'` below is caused by the fact that since #6278 tools return None when there is not stdout. You can get consistent output with `consistent_return_value=True` added to the Tools object creation. The default is "(most likely) expected behavior" while you need "consistent, predictable behavior" which is what the new parameter is for. I hope this makes sense.

Reply:
Thanks, that was the missing link for me...